app/views/welcome/_resources.md

```diff
@@ -51,6 +51,7 @@ successfully pair-programming beyond your office.
   that allows "developers around the world to edit the same code and chat
   together in real-time". You can sign in with your github account and clone
   repos straight into it.
+- [CoScreen](https://coscreen.co) Make your desktop a shared remote collaborative pair-programming workspace. Share and interact with multiple applications between multiple users, with integrated non-intrusive audio and video chat. Bring your own IDE, mobile emulators, design, or debug tools.
```

Comment:
should be marked as freemium or paid or some such.

What's your relationship?

Reply:
I'll add in freemium, I'm the CTO.